### ISSUE TITLE
Interesting popularisation article about alternative (ranking based) voting systems and their flaws

### ISSUE THREAD

**Issue by jibe-b** [democratech/LaPrimaire#44]
http://ncase.me/ballot/

**Comment by ghost** [comment]
Merci pour cet excellent article, qui remet bien les pendules à l'heure.
C'est juste dommage que le simulateur n'inclue pas le jugement majoritaire :-/

**Comment by jibe-b** [comment]
Je crois que l'auteur, en plus d'être rigoureux et doué est du genre à apprécier la coopération.

Je vais lui proposer le jugement majoritaire, qui mériterait en effet d'être dans la liste.

Est-ce que si je lui indique ce répo tu répondrais à ses questions @GBodin (ou qui veut répondre, bien sûr) ?

**Comment by ghost** [comment]
Oui, bien sûr ! Ce serait tellement bien de pouvoir utiliser son outil pour illustrer les arguments pour et contre le jugement majoritaire, j'y contribuerai bien volontiers si je peux aider.

**Comment by xDEADC0DE** [comment]
Bonjour, j'ai participé à la traduction française de la page de ncase (elle est ici pour le moment : https://xdeadc0de.github.io/ballot-fr/ ). Je ne connaissais pas le jugement majoritaire, mea culpa.

Ncase ne souhaite pas faire de changements sur la page après l'avoir publiée, mais peut être qu'on pourrait faire une suite, en parlant du jugement majoritaire, en illustrant avec les résultats de l'élection présidentielle par exemple.
